packages/appChrome/components/HeaderBar.tsx

```diff
@@ -1,15 +1,68 @@
 import * as React from "react";
 import { cx } from "emotion";
-import { darkMode, flex, padding } from "../../shared/styles/styleUtils";
+import {
+  background,
+  darkMode,
+  flex,
+  padding
+} from "../../shared/styles/styleUtils";
 import { headerBar, appChromeInsetContent } from "../style";
 
 export interface HeaderProps {
+  backgroundColor?: string;
   children: React.ReactElement<HTMLElement> | string;
+  paddingBottom?:
+    | "xxs"
+    | "xs"
+    | "s"
+    | "m"
+    | "l"
+    | "xl"
+    | "xxl"
+    | "none"
+    | undefined;
+  paddingLeft?:
+    | "xxs"
+    | "xs"
+    | "s"
+    | "m"
+    | "l"
+    | "xl"
+    | "xxl"
+    | "none"
+    | undefined;
+  paddingRight?:
+    | "xxs"
+    | "xs"
+    | "s"
+    | "m"
+    | "l"
+    | "xl"
+    | "xxl"
+    | "none"
+    | undefined;
+  paddingTop?:
+    | "xxs"
+    | "xs"
+    | "s"
+    | "m"
+    | "l"
+    | "xl"
+    | "xxl"
+    | "none"
+    | undefined;
 }
 
 class Header extends React.PureComponent<HeaderProps, {}> {
   public render() {
-    const { children } = this.props;
+    const {
+      children,
+      backgroundColor,
+      paddingBottom,
```

Comment:
Deconstructing might make the defaults easier to visually grasp

```suggestion
      paddingBottom = "xs",
```